android/build.gradle

```diff
@@ -1,60 +1,58 @@
-// There is no version defined on the package.
-// Since Capacitor 5 the minimum Java version was altered to version 18 so we will use this information to define if
-// the project is running on Capacitor 5 or higher.
-def cap5OrHigher =  rootProject.project(':capacitor-android').android.compileOptions.sourceCompatibility >= JavaVersion.VERSION_17
+def safeExtGet(prop, fallback) {
+  rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
+def getCapacitorCompileSdkVersion() {
+  return  Integer.parseInt(
+    rootProject.project(':capacitor-android').android.compileSdkVersion
+      .replaceAll(/\D+/, ''))
+}
+
+def GetCapacitorVersion() {
+  def compileVersion =  getCapacitorCompileSdkVersion()
+  if (compileVersion == 34) {
+    return 6
+  }
+  else if (compileVersion == 33) {
+    return 5
+  }
+  else if (compileVersion == 32) {
+    return 4
+  }
+  else {
+    throw new GradleException("The version of Capacitor is not supported by Sentry, CODE: " + compileVersion)
```

Comment:
This seems like a breaking change.

Did the SDK build with Capacitor 3 before?

Reply:
Or does the case for 4 covers 3 as well?

Reply:
Nice catch, I'll fix the case for Capacitor 3